### DISCUSSION TITLE
(초안) 2월 28일

### DISCUSSION THREAD

**Discussion by rhkdgns95** [dennis2311/Eagloo-front#4] [General]
**2021.02.28 (일)**
- 번들러 교체 배포 github action 배포 적용.
- 소켓 채널명 상수화.
- JWT도입 (서버측과 클라이언트측의 식별을 위한 key-value 정의)
- 서버측에 필요한 이벤트를 API를 전달. (Fetch요청을 해서 처리하는 부분이 필요함)

**Next Steps**
- 중간점검 : 3월 7~9일
- 최종점검일 : 3월 29일 (월)

**Comment by rhkdgns95** [comment]
@dennis2311

**Comment by dennis2311** [comment]
> 채널명 상수화 & fetch 요청
- [ ] 로그인 (POST, 요청이후 JWT발급)
- [ ] 채팅방 참여 (소켓 : CONNECT_ROOM)
- [ ] 다른유저 참여 (소켓 : NEW_PEER_CONNECT_ROOM)
- [ ] 채팅방 나가기 (소켓 : DISCONNECT_ROOM)
- [ ] 개인방 생성 (POST : 향후 조정)
- [ ] 채팅 (소켓 : 향후 조정)
